CountOnMe/Controller/Delegate+Extension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Delegate+Extension.swift
+//  CountOnMe
+//
+//  Created by Redouane on 08/03/2024.
+//  Copyright © 2024 Vincent Saluzzo. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+// MARK: - Extension CalculatorManagerDelegate
+
+extension ViewController: CalculatorManagerDelegate {
+    func showError(_ error: CalculationError) {
+        let alert: UIAlertController = .init(title: "Unvalid operation", message: error.message, preferredStyle: .alert)
```

Comment:
**suggestion (blocking)**: There is a typo in the title. Also you can write it like this for additional clarity.

```suggestion
        let alert = UIAlertController(title: "Invalid operation", message: error.message, preferredStyle: .alert)
```